assets/scss/variables/index.scss

```
@@ -7,6 +7,7 @@ $gray: #567fa8;
 $yellow: #ffdb00;
 $bg-sidebar: hsla(0, 0%, 100%, 0.05);
 $border-primary: hsla(0, 0%, 100%, 0.1);
+$hover-text: #c273ed;
```

Đặt tên màu, rồi dùng màu cho hover
```suggestion
$tenMau: #c273ed;
```